valuable/src/value.rs

```diff
@@ -690,6 +712,34 @@ macro_rules! convert {
                     _ => None,
                 }
             }
+
+            /// Return a `&dyn Display` representation of `self`, if possible.
+            ///
+            /// # Examples
+            ///
+            /// ```
+            /// use std::fmt;
+            /// use valuable::{Value, Valuable};
+            ///
+            /// struct Meters(u32);
+            ///
+            /// impl fmt::Display for Meters {
+            ///     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+            ///         write!(f, "{}m", self.0)
+            ///     }
+            /// }
+            ///
+            /// let meters = Meters(5);
+            ///
+            /// assert!(Value::Display(&meters).as_display().is_some());
+            /// assert!(Value::Bool(true).as_display().is_none());
+            /// ```
+            pub fn as_display(&self) -> Option<&dyn fmt::Display> {
+                match *self {
+                    Value::Display(v) => Some(v),
+                    _ => None,
```

Comment:
hmm, not sure if this is correct. most of the primitive `Value` variants are _also_ types that implement `Display ` (e.g. strings, numbers, boolean values, and even `Path`s and `Error`s), so I think we should return them here, as well. i think it's only structured values (like `Value::Structable`, `Value::Tupleable`, etc) that can't return a `Display` trait object here.